src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/FlexTicketDiscountInfo.tsx

```diff
@@ -30,13 +30,14 @@ type Props = {
 
 export const FlexTicketDiscountInfo = ({userProfiles, style}: Props) => {
   const {t, language} = useTranslation();
-  const [expanded, setExpanded] = useState(true);
+  const [expanded, setExpanded] = useState(false);
   const styles = useStyles();
   const {appTexts} = useFirestoreConfiguration();
   const {flex_ticket_url} = useRemoteConfig();
 
-  if (!userProfiles.some((u) => u.offer.flex_discount_ladder)) return null;
-  
+
+  if (!userProfiles.every((u) => u.offer.flex_discount_ladder)) return null;
```

Comment:
Since there is no discount when purchasing for others, also return null if `isOnBehalfOfToggle` is `true`?

Reply:
On-behalf-of will be done in another PR, there's more to do than just hiding the flex discount information. Mainly I need to handle the display of price, for example, we have to show original price instead of discounted price when buying for others, also need to check how the behavior goes with the new system, etc. That's why I didn't include the reference screen about the on-behalf-of in this PR.